mono/arch/s390x/s390x-codegen.h

```diff
@@ -268,7 +268,7 @@ typedef struct {
 	char	xx : 4;
 	char	r3 : 4;
 	char	r4 : 4;
-} __attribute__ ((packed)) RRD_Format;
+} __attribute__ ((__packed__)) RRD_Format;
```

Comment:
I just PR’d exactly what Flint suggested, to see which changes were wanted.

Their reasoning is that identifiers containing `__` are reserved, so the underscore-adorned attribute should always work even in the presence of macros such as `#define error abort` or `#define printf g_printf`. As far as I can tell, the embedding API is currently the only place this might matter for Mono.

Clang also appears to accept this syntax—I was under the impression that their policy is to support all GNU attributes and extensions, but I can’t seem to find a citation for that.